src/parameters.ts

```diff
@@ -1,7 +1,7 @@
 import { camelCase } from "change-case";
 
-const parameterize = (parametersArray) => {
-  return new Set(parametersArray.map(camelCase));
+const parameterize = (parametersArray: string[]) => {
+  return new Set(parametersArray.map((param) => camelCase(param)));
```

Comment:
We need to bind an anonymous function here because camelCase takes a second argument, and map is passing one, but they are of disparate types. Another bug caught by Typescript.